types/got/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for got 7.1
+// Type definitions for got 8.3
 // Project: https://github.com/sindresorhus/got#readme
 // Definitions by: BendingBender <https://github.com/BendingBender>
 //                 Linus Unnebäck <https://github.com/LinusU>
```

Comment:
If you want to, you can add yourself to this list. That way you'll be able to help review pull requests to this package in the future ☺️